### PR TITLE
Mobileapps 1978

### PR DESCRIPTION
**Pull request by aman-alfresco**
- added selection and deselection of rows
- unit test cases

### CODE REVIEW

File: browse/src/main/kotlin/com/alfresco/content/browse/BrowseViewModel.kt [hunk 1]
```diff
@@ -295,11 +302,32 @@ class BrowseViewModel(
...
+        println("BrowseViewModel.toggleSelection 0 == ${entries.size}")
+        val updatedEntries = entries.map {
+            println("BrowseViewModel.toggleSelection 1 == $it")
+            if (it.id == entry.id && it.type != Entry.Type.GROUP) {
+                it.copy(isSelectedForMultiSelection = !it.isSelectedForMultiSelection)
+            } else {
+                it
+            }
+        }
+        println("BrowseViewModel.toggleSelection 2 == ${updatedEntries.size}")
```
Comment (gl-lovekesh) [inline browse/src/main/kotlin/com/alfresco/content/browse/BrowseViewModel.kt:315]: remove the Print statement.

File: move/src/main/java/com/alfresco/content/move/BrowseMoveFragment.kt [hunk 2]
```diff
@@ -108,4 +108,7 @@ class BrowseMoveFragment : ListFragment<BrowseViewModel, BrowseViewState>(R.layo
             findNavController().navigateToFolder(entry, state.moveId)
         }
     }
+
+    override fun onItemLongClicked(entry: Entry) {
```
Comment (gl-lovekesh) [inline move/src/main/java/com/alfresco/content/move/BrowseMoveFragment.kt:112]: Remove empty function.

**Review by gl-lovekesh — CHANGES_REQUESTED** [review]

File: browse/src/main/kotlin/com/alfresco/content/browse/BrowseViewModel.kt (hunk 1 above)
Comment (aman-alfresco) [reply browse/src/main/kotlin/com/alfresco/content/browse/BrowseViewModel.kt]: Done

File: move/src/main/java/com/alfresco/content/move/BrowseMoveFragment.kt (hunk 2 above)
Comment (aman-alfresco) [reply move/src/main/java/com/alfresco/content/move/BrowseMoveFragment.kt]: Done

**Review by gl-lovekesh — APPROVED** [review]

**Pull request merged**